commit 1e85efb4b13fea167cd2f805c62b40053db67ce0
Author: Yuki Schlarb <erin-dev@ninetailed.ninja>
Date:   Thu Sep 22 22:16:19 2022 +0200

    Use Python 3.6+ compatible typing annotation syntax in “lutris/util/discord/client.py”, so that Lutris does not crash on startup on Python 3.9 and below if the `pypresence` package is importable

diff --git a/lutris/util/discord/client.py b/lutris/util/discord/client.py
index ed6711c8..b17183f5 100644
--- a/lutris/util/discord/client.py
+++ b/lutris/util/discord/client.py
@@ -1,10 +1,12 @@
+import typing as ty
+
 from pypresence import Presence
 
 from lutris.util.discord.base import DiscordRichPresenceBase
 
 
 class DiscordRichPresenceClient(DiscordRichPresenceBase):
-    rpc: Presence | None  # Presence Object
+    rpc: ty.Optional[Presence]  # Presence Object
 
     def __init__(self):
         self.playing = None
